Sources/Web3Modal/Core/Web3ModalClient.swift

```diff
@@ -235,4 +332,14 @@ public class Web3ModalClient {
             UIApplication.shared.open(url, completionHandler: nil)
         }
     }
+    
+    @discardableResult
+    public func handleDeeplink(_ url: URL) -> Bool {
```

Comment:
it used for coinbase only?

Reply:
Yes for now only coinbase uses deeplinks.